substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ProxyConfigurationParser.java

```diff
@@ -72,4 +87,9 @@ private void parseInterfaceList(List<?> data) {
         }
     }
 
+    private void parseWithPredicatedConfig(Map<String, Object> proxyConfigObject) {
+        checkAttributes(proxyConfigObject, "proxy descriptor object", Collections.singleton("interfaces"));
+        parseInterfaceList(asList(proxyConfigObject.get("interfaces"), "interfaces must be an array of fully qualified interface names"));
```

Comment:
Better quote `"interfaces" must be an array of ...`

Reply:
Agreed, I've changed it to `"interfaces"` in the message